### PR TITLE
Arrays now can store objects other than Jastor classes 

### PR DESCRIPTION
**Pull request by dadalar**
Arrays can store NSString, NSNumber, NSDictionary classes.
Also, cocoapods version is updated. Please add a tag with 0.2.1 version.

### CODE REVIEW

**Comment by elado** [comment]
Thanks. Can you please add tests?

**Comment by dadalar** [comment]
I did. Thanks.

File: jastor.podspec [hunk 1]
```diff
@@ -1,10 +1,10 @@
 Pod::Spec.new do |s|
   s.name         = "jastor"
-  s.version      = "0.2.0"
+  s.version      = "0.2.1"
   s.summary      = "Auto translates NSDictionary to instances of Objective-C classes, supporting nested types and arrays."
   s.homepage     = "https://github.com/elado/jastor"
   s.license      = { :type => 'MIT', :file => 'LICENSE' }
   s.author       = { "Elad Ossadon" => "elad@ossadon.com" }
-  s.source       = { :git => "https://github.com/elado/jastor.git", :tag => "0.2.0" }
+  s.source       = { :git => "https://github.com/dadalar/jastor.git", :tag => "0.2.1" }
```
Comment (elado) [inline jastor.podspec]: Please revert to the original source of the pod. I'll create a new tag.

**Comment by dadalar** [comment]
done

**Pull request merged**